tests/common/test_request_xml.py

```diff
@@ -37,7 +39,7 @@ def commit():
     refresh_dict = {}
     mo_dict = to_commit
     if not mo_dict:
-        print "No Mo to be Committed"
+        print("No Mo to be Committed")
```

Comment:
May be we should extract print out to a utils file. And handle Python 2 vs Python 3 logic there. Applies to all other print statements too.